<!-- connect_tf_trees.launch
Publish tf between fisheye -> virtual tags and base_link -> ground truth tags
to conenct everything to one tf tree -->

<launch>
  <arg name="base_frame" default="base_link" />
  <arg name="camera_frame1" default="fisheye" />
  <arg name="camera_frame2" default="cam0" />
  <arg name="timeout" default="10" />
  <arg name="tag_frames" default="['tag_584', 'tag_582', 'tag_580',
                                   'tag_579', 'tag_578', 'tag_577',
                                   'tag_9', 'tag_5', 'tag_7']" />

  <param name="/use_sim_time" value="true"/>

  <node pkg="joint_calibration" type="virtual_tfs.py"
        name="virtual_tfs" output="screen"/>

</launch>
